ee/cis/macos-13/cis-policy-queries.yml

```diff
@@ -103,6 +103,64 @@ spec:
 ---
 apiVersion: v1
 kind: policy
+spec:
+  name: CIS - Ensure iCloud Keychain is enabled (if your org policy is to enable it) (MDM Required)
+  platforms: macOS
+  platform: darwin
+  description: |
+    The iCloud keychain is Apple's password manager that works with macOS and iOS. The capability allows users to store passwords in either iOS or macOS for use in Safari on both platforms and other iOS-integrated applications. The most pervasive use is driven by iOS use rather than macOS. The passwords stored in a macOS keychain on an Enterprise-managed computer could be stored in Apple's cloud and then be available on a personal computer using the same account. The stored passwords could be for organizational as well as for personal accounts.
+    If passwords are no longer being used as organizational tokens, they are not in scope for iCloud keychain storage.
+    Rationale:
+    Ensure that the iCloud keychain is used consistently with organizational requirements.
+  resolution: |
+    The administrator should configure this via MDM profile.
+    Create or edit a configuration profile with the following information:
+      1. The PayloadType string is com.apple.applicationaccess.
+      2. The key to include is allowCloudKeychainSync.
+      3. The key must be set to <true/>.
+  query: |
+    SELECT 1 WHERE NOT EXISTS (
+        SELECT 1 FROM managed_policies WHERE 
+            domain='com.apple.applicationaccess' 
+            AND 
+            name='allowCloudKeychainSync' 
+            AND (value = 0 OR value = 'false') 
+    );
```

Comment:
For these "manual" checks we should have one check as usual but with one small difference:
```sql
SELECT 1 FROM managed_policies
WHERE domain='com.apple.applicationaccess' AND name='allowCloudKeychainSync' 
 ```
The above would be checking that the administrator has set the setting via a profile. We (Fleet) do not care about its value (and cannot enforce one actually), we only want to check that the administrator has audited this check and decided their settings for their organization.

Reply:
That's correct.
I'll fix it.